components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/resources/GroupResource.java

```diff
@@ -306,24 +431,28 @@ public Response patchGroup(@PathParam(SCIMConstants.CommonSchemaConstants.ID) St
     private Response processRequest(final Map<String, String> requestAttributes) {
 
         String id = requestAttributes.get(SCIMProviderConstants.ID);
-        String userName = requestAttributes.get(SCIMProviderConstants.AUTHORIZATION);
```

Comment:
Why did we remove this?

Reply:
this is not used anywhere. And it was suggested by idea